### DISCUSSION TITLE
Elsa v3 release dates?

### DISCUSSION THREAD

**Discussion by NrujalSharma** [elsa-workflows/elsa-core#3677] [General]
What are the tentative release dates for v3 with designer ready to use with Angular/React?

**Comment by sfmskywalker** [comment]
It looks like an RC might happen sometime towards the end of March. We're currently working on the workflow journal and input/output support of activities, allowing you to create workflows in the designer that can then be used as composite activities in other workflows. Once that is done, we need a bit of time to complete the initial version of the documentation.

**Reply by NrujalSharma** [reply]
Thanks for the update @sfmskywalker 🙌🏽

**Reply by truthz03** [reply]
Hi, is there already a more specific date for when version 3 will be available as a NuGet package?

**Reply by sfmskywalker** [reply]
I'm afraid no specific date yet. Elsa 3 is still going through a lot of testing, from which various issues arise. Once this settles down, it might be a little bit easier to make a prediction, but I'm thinking (hoping?) it might happen somewhere in April.

**Reply by mrubelmann** [reply]
I know, I hate it when people ask me for dates too, but...  is it almost ready?  I want v3 so, so, so badly!  :)

**Reply by sfmskywalker** [reply]
Completely understand 😄  The date for RC is May 31. After that, I am hoping to do a 3.0 GA release one or two months later, depending on feedback from users.
